docs/contracts/v4/guides/04-hooks/setup.mdx

```diff
@@ -0,0 +1,223 @@
+---
+title:  Set Up Your Local Environment
+---
+
+Before writing our hook let's first have our local environment properly configured e.g. deploying pool manager, utility routers and test tokens. 
+
+At the end of this guide you’ll have a development environment set up that you can use to build the rest of the examples in the Guides section of the docs, or start your own integration project!
+
+To get you started as quickly as possible for building Uniswap v4 hooks, we have provided the `Quick Start` section below where you can clone a boilerplate and get building. To start from scratch and learn the underlying concepts, jump to the `Start from Scratch` section.
```

Comment:
let's remove 2nd person language ("you"). its not here, but found in other places too

Reply:
Got it, since I am referencing the "set up your local environment" part from v3 docs

Reply:
so we shouldn't use first or second person on most parts of v4 doc right?